libminifi/include/io/BaseStream.h

```diff
@@ -175,13 +174,12 @@ class BaseStream : public DataStream, public Serializable {
    **/
   virtual int read(uint64_t &value, bool is_little_endian = EndiannessCheck::IS_LITTLE);
 
-  virtual const uint64_t getSize() const {
+  virtual uint64_t getSize() const {
```

Comment:
This breaks API.
The original one does not make sense, and I agree with this change, but we can't break the API in a minor release, especially an API very likely used in the wild.